app/controllers/submit.js

```diff
@@ -350,6 +376,13 @@ export default Ember.Controller.extend(BasicsValidations, NodeActionsMixin, Tagg
     }),
 
     actions: {
+        editLicense(license, validates) {
```

Comment:
Will use Ember.run.once on _setNodeLicense to ensure that the bouncing doesn't happen.